### ISSUE TITLE
suggestion : Configs

### ISSUE THREAD

**Issue by gogoghouha** [Ted80-Minecraft-Mods/Realistic-World-Gen#20]
Could you add other configurations such as
- biomes weights ?
- disable biomes to gen ?

keep the good work ! ;)

**Comment by Dream-Master** [comment]
Yes this would be nice better control which biome is enabled because i didn't see any Thaumcraft Biomes yet. To make the size of any Biome configurable would be nice too.
